react/features/base/settings/reducer.js

```diff
@@ -125,6 +126,7 @@ function _initSettings(featureState) {
     // jibri, and remove the old settings.js values.
     const savedDisplayName = jitsiLocalStorage.getItem('displayname');
     const savedEmail = jitsiLocalStorage.getItem('email');
+    const userSelectedSkipPrejoin = jitsiLocalStorage.getItem('userSelectedSkipPrejoin');
```

Comment:
No need for this. Jibri is now ready to override the value in config.js

Reply:
Removed it.